### PR TITLE
lint unnecessary_const on samples

### PR DESCRIPTION
**Pull request by a14n**

### CODE REVIEW

**Comment by a14n** [comment]
@Hixie

**Comment by a14n** [comment]
@Hixie do you receive notifications from this PR?

**Comment by Hixie** [comment]
@a14n I do, but I get so many notifications that they all get drowned out and I end up missing them.

File: dev/bots/analyze-sample-code.dart [hunk 1]
```diff
@@ -212,6 +213,11 @@ dependencies:
     sdk: flutter
   flutter_test:
     sdk: flutter
+''');
+    analysisOptions.writeAsStringSync('''
+linter:
+  rules:
+    - unnecessary_const
```
Comment (Hixie) [inline dev/bots/analyze-sample-code.dart]: we probably want to apply many more lints, too... like maybe all the lints from the analysis_options file.

**Comment by Hixie** [comment]
LGTM

File: dev/bots/analyze-sample-code.dart (hunk 1 above)
Comment (a14n) [reply dev/bots/analyze-sample-code.dart]: Yes, this PR is just the first step ;) `unnecessary_new` will come quickly.

**Pull request merged**